.env.example

```
@@ -3,4 +3,14 @@ CLIENT_SECRET=<YOUR REDDIT APP CLIENT SECRET>
 REDDIT_REDIRECT=<http://localhost:3000/api/auth/callback/reddit>
 NEXTAUTH_SECRET=generate one, example with >openssl rand -base64 32
 NEXTAUTH_URL=http://localhost:3000
-SIGNING_PRIVATE_KEY=generate one, example with >jose newkey -s 256 -t oct -a HS512
\ No newline at end of file
+SIGNING_PRIVATE_KEY=generate one, example with >jose newkey -s 256 -t oct -a HS512
```

dotenv-linter (3.3.0)

[warning] [ValueWithoutQuotes] This value needs to be surrounded in quotes

(ValueWithoutQuotes)